HISTORY.rst

```diff
@@ -2,15 +2,35 @@
 History
 =======
 
+3.51.0 (2022-01-20)
+-------------------
+
+Changes
+
+* The ``cci flow doc`` command now only includes CumulusCI's standard flows unless the ``--project`` option is specified. (#3033)
+* The ``run_tests`` task now has a ``required_per_class_code_coverage_percent`` which ensures that every class in your project meets the code coverage level specified. (#3027)
+* Marketing Cloud tasks:
+
+  * Added the ``marketing_cloud_get_user_info`` task to retrieve user information from the Marketing Cloud REST API ``userinfo`` endpoint. (#3039)
+  * The ``marketing_cloud_create_user`` task now creates an unlocked user with a notification email address so that it is possible for the user to log in. This task also has a new option, ``activate_if_existing``, which can be set to True to ensure that if the user already exists in an inactive state, it will be activated. (#3040)
```

Comment:
```suggestion
  * The ``marketing_cloud_create_user`` task now creates an unlocked user with a notification email address so that it is possible for the user to log in. This task also has a new option, ``activate_if_existing``, which can be set to ``true`` to ensure that if the user already exists in an inactive state, it will be activated. (#3040)
```